### PR TITLE
Modify messages for splitting army window

### PR DESCRIPTION
**Pull request by Lumary2**
Happy Hacktoberfest :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Windows
 
 MacOS and Linux
 -------------------
-- open `script/macos` or `script/linux` directory depending on your OS and run **install_sdl_1.sh** or **install_sdl_2.sh** file. For MacOS we recommend to run SDL 2 as latest versions of MacOS does not support SDL 1 fully.
+- open `script/macos` or `script/linux` directory depending on your OS and run **install_sdl_1.sh** or **install_sdl_2.sh** file.  For Arch based distros you need to install SDL with the command `sudo pacman -S sdl sdl_mixer` instead. For MacOS we recommend to run SDL 2 as latest versions of MacOS does not support SDL 1 fully.
```
Comment (ihhub) [inline README.md:43]: Hi @Lumary2 , could you please revert changes in this file as we follow the rule of having only related changes in pull request?

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Lumary2, could you please make the requested change or discuss it?

File: README.md (hunk 1 above)
Comment (Lumary2) [reply README.md:43]: Hi @ihhub You can close this pr and I did a new one without the readme change.

**Comment by ihhub** [comment]
Closing this pull request by author's request.

**Pull request closed without merging**